src/test/groovy/com/florianmski/suncalc/SunCalcSpec.groovy

```diff
@@ -61,16 +63,15 @@ class SunCalcSpec extends spock.lang.Specification {
 
     }
 
-    @Ignore("original Moon test code ported, but not yet fully tested")
+    @Ignore("original Moon Java test code ported, but not yet fully tested")
```

Comment:
Well giving it another look it seems it is not asserting anything? I'm not familiar with Spock so I might have missed something though. That should be removed then